src/components/scenes/FioAddressDetailsScene.js

```diff
@@ -42,6 +42,11 @@ export class FioAddressDetailsScene extends Component<Props, State> {
     Actions[Constants.FIO_CONNECT_TO_WALLETS]({ fioAddressName })
   }
```

Comment:
This scene is going away. We are submitting designs to put the renew button on the wallet connect screen.